elife/gearman-client-php.sls

```diff
@@ -39,9 +44,51 @@ php-ppa-gearman:
             - test -e /etc/apt/sources.list.d/ondrej-ubuntu-pkg-gearman-xenial.list
         - require_in:
             - pkg: gearman-php-extension
-{% endif %}
 
 gearman-php-extension:
     pkg.installed:
         - pkgs:
             - php-gearman
+
+{% elif salt['grains.get']('osrelease') == '18.04' %}
+
+php-ppa-gearman:
+    pkg.purged:
+        - pkgs:
+            - php-gearman
+        - require_in:
+            - gearman-php-extension
```

Comment:
IIRC for `require_in` you need to specify `- git: gearman-php-extension` with the state type or it fails silently; I almost never use it in favor of `require` from the other side

Reply:
> I almost never use it in favor of require from the other side

I don't see anything wrong with using it, but it is awfully frustrating when it fails silently because you forgot to do the fully qualified state name thing

Reply:
fixed